<launch>
  <node name="cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="image_proc" pkg="image_proc" type="image_proc" ns="cam"/>

  <node name="face_detection" pkg="opencv_apps" type="face_detection" args="image:=/cam/image_rect">
    <remap from="facedetection/faces" to="/face"/>
  </node>

</launch>
